smoke-tests/src/test/java/com/splunk/opentelemetry/TomcatSmokeTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry;
+
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+import java.io.IOException;
+import java.util.stream.Stream;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class TomcatSmokeTest extends AppServerTest {
+
+  public static final ExpectedServerAttributes TOMCAT7_SERVER_ATTRIBUTES =
+      new TomcatAttributes("7.0.107.0");
```

Comment:
How fragile are these versions? I believe we build docker images with "current latest for this major version".

Reply:
There are multiple scenarios to this :) If we publish our matrix images and will not rebuild them - then the version won't change :) If we build every time before tests - they most probably will break on next released version.

I see 2 potential solutions - fix the version in the matrix or use partial matching in assertions. I'd vote for fixing versions in the matrix.

Reply:
As discussed in slack, versions will be fixed while upstreaming the matrix for fully open-source app-servers.